libs/apigateway-lambda-proxy/src/lib/metric-alarms.ts

```diff
@@ -155,6 +155,7 @@ export class ErrorRate5xxAlarm extends pulumi.ComponentResource {
         new aws.cloudwatch.MetricAlarm(
             resourceName,
             {
+                period: 300,
```

Comment:
When metricQueries isn't used, period is required.

Reply:
Does this mean it will require 10 minutes to alarm (as it needs 2 evaluation periods, and each period is 5 minutes?

Reply:
yep

Reply:
I'm going to reduce the period to 60 -- that's the resolution I'm seeing in the-game-prod.

Reply:
(done)